### ISSUE TITLE
Make it possible for people to comment on moves via Twitter

### ISSUE THREAD

**Issue by slifty** [BadIdeaFactory/overengineered-diplomacy#1]
People should be able to comment on specific moves within the game via twitter, allowing crowd contributions to the games overall narrative

**Comment by knowtheory** [comment]
Yeah it should probably have two views, an event log view of just the moves (with tweets or whatever interspersed) and a map view where you can watch control change hand over the years.

But that'd provide a location into which tweets or whatever can be inserted or moves linked to.

**Comment by slifty** [comment]
Also not just comments -- we can use encrypted data in the tweet to allow the user to specify things like round, location, and style of the message so it can be rendered in the most appropriate way.
